index.d.ts

```diff
@@ -18,6 +19,8 @@ export interface Options {
   testNamePattern?: string;
   testFileNamePattern?: string;
   reporters?: string[];
+  // extra args if specified will be appended to auto arguments
+  extraArgs?: string[];

```

Comment:
I would call it something like `autoArgs`, since they don't get used if `args` is set.

Reply:
good point, I guess we could consolidate these 2 into 1, like:
```
interface RunArgs {
    args: string[];
    replace?: boolean; 
}
```